examples/bc5cdr.py

```diff
@@ -58,65 +65,110 @@
                 {
-                    "passages": datasets.Sequence(
+                    "passages": [
                         {
-                            "pmid": datasets.Value("string"),
+                            "document_id": datasets.Value("string"),
                             "type": datasets.Value("string"),
                             "text": datasets.Value("string"),
-                            "entities": datasets.Sequence(
+                            "entities": [
                                 {
-                                    "offsets": datasets.Sequence(
-                                        [datasets.Value("int32")]
-                                    ),
-                                    "text": datasets.Value("string"),
+                                    "id": datasets.Value("string"),
+                                    "offsets": [[datasets.Value("int32")]],
+                                    "text": [datasets.Value("string")],
                                     "type": datasets.Value("string"),
-                                    "entity_id": datasets.Value("string"),
+                                    "normalized": [
+                                        {
+                                            "db_name": datasets.Value("string"),
+                                            "db_id": datasets.Value("string"),
+                                        }
+                                    ],
                                 }
-                            ),
-                            "relations": datasets.Sequence(
+                            ],
+                            "relations": [
                                 {
-                                    'relation': datasets.Value("string"),
-                                    'chemical': datasets.Value("string"),
-                                    'disease': datasets.Value("string"),
+                                    "id": datasets.Value("string"),
+                                    "type": datasets.Value("string"),
+                                    "arg1_id": datasets.Value("string"),
+                                    "arg2_id": datasets.Value("string"),
                                 }
-                            ),
+                            ],
                         }
-                    )
+                    ]
+                }
+            )
+
+        elif self.config.name == "bigbio":
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "document_id": datasets.Value("string"),
+                    "passages": [
+                        {
+                            "id": datasets.Value("string"),
+                            "type": datasets.Value("string"),
+                            "text": datasets.Value("string"),
+                            "offsets": datasets.Sequence([datasets.Value("int32")]),
```

Comment:
```suggestion
                            "offsets": datasets.Sequence(datasets.Value("int32")),
```

Reply:
Using`datasets.Sequence([datasets.Value("int32")])` for consistency with entities, etc, per slack conversation